vsintegration/tests/FSharp.Editor.Tests/CodeFixes/PrefixUnusedValueTests.fs

```diff
@@ -57,25 +57,4 @@ let f() =
 
     Assert.Equal(expected, actual)
 
-[<Fact>]
```

Comment:
See 13c28405020d8dad69b231afc3526a5635901e70. That's because the diagnostics are now being filtered down to fixable ones, although that can be reverted if needed.